src/Handler/StreamHandler.php

```diff
@@ -304,27 +311,28 @@ private function createStream(RequestInterface $request, array $options)
         }
 
         // Microsoft NTLM authentication only supported with curl handler
-        if (isset($options['auth'])
-            && \is_array($options['auth'])
-            && isset($options['auth'][2])
-            && 'ntlm' == $options['auth'][2]
+        if (isset($options['auth'][2]) && 'ntlm' === $options['auth'][2]
         ) {
```

Comment:
This parenthesis should move up.

Reply:
Interesting that cs-fixer did not catch this one. Fixed in 44148c15357f5601487877f3d8bf8a97a8c4b1af